ui/tests/acceptance/config-ui/login-settings-test.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { click, visit, currentRouteName } from '@ember/test-helpers';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { runCmd } from 'vault/tests/helpers/commands';
+
+module('Acceptance | Enterprise | config-ui/login-settings', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    const version = this.owner.lookup('service:version');
+    version.type = 'enterprise';
```

Comment:
this can be removed since the "Enterprise" in the title means this test will be filtered out on CE runs (which also means before merging we'll need to double check enterprise tests pass locally)